lib/board.rb

```diff
@@ -2,7 +2,6 @@
 
 # Board class
 class Board
-  attr_reader :possible_wins
   attr_accessor :board
 
   def initialize
```

Comment:
Great work guys, running a test on `board_spec`, you currently have arr with 9 items in `board.rb` - line 8, once i change the arr to items of 8, the test case failed as expected :+1: , throwing this errors messages :point_right: `# Board#initialize should make an array of 9 items`
But for `game.spec` spec is not failing after i making some changes in `game.rb`
_example:_ 
- i made a change on line 32 (changing `@input` to something else, test still passess)
- also made a change on line 43 ( `is_a? Integer` i changed it to `is_a? symbol`) test sill passes)
kindly work on all the test you wrote for the methods inside `game.rb` except `switch_player`